### PR TITLE
CDM-43: Team Element & TeamElementRow Components

### PR DESCRIPTION
**Pull request by mariasbl**
- **Web** 
image

- **Mobile**
image

### CODE REVIEW

File: fe/app/components/TeamElement/TeamElement.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import ImageElement from "../ImageElement/ImageElement";
+import { TeamElementProps } from "./TeamElement.models";
+
+const TeamElement = ({ image, title, subtitle }: TeamElementProps) => {
+  return (
+    <div className="flex flex-col items-center">
+      <ImageElement
+        {...image}
+        //TODO: replace rounded classname with new flag rounded
+        className="rounded-[50%] overflow-hidden h-[80px] w-[80px] md:h-[200px] md:w-[200px]"
```
Comment (danielaprado) [inline fe/app/components/TeamElement/TeamElement.tsx:11]: Tailwind has the 80px value 🙌  `h-20` and `w-20`

File: fe/app/components/TeamElementRow/TeamElementRow.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    };
+
+    setContent(splitArray());
+  }, [limitPerRow, teamElements]);
+
+  return (
+    <div className="flex flex-col gap-8 md:gap-16">
+      {content.map((group, i) => (
+        <div className="flex" key={i}>
+          <Divider className="h-[2px] w-full bg-white mr-4 mt-10 md:mr-16 md:mt-[100px]" />
```
Comment (danielaprado) [inline fe/app/components/TeamElementRow/TeamElementRow.tsx:31]: And also has 2px, it's `h-0.5`

File: fe/app/components/TeamElementRow/TeamElementRow.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <div className="flex gap-8 md:gap-16">
+            {group.map((teamElement) => (
+              <TeamElement key={i} {...teamElement} />
+            ))}
+          </div>
+          <Divider className="h-[2px] w-full bg-white ml-4 mt-10 md:ml-16 md:mt-[100px]" />
```
Comment (danielaprado) [inline fe/app/components/TeamElementRow/TeamElementRow.tsx:37]: Same has the previous one!

File: fe/app/components/TeamElementRow/TeamElementRow.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { TeamElementProps } from "../TeamElement/TeamElement.models";
+import TeamElement from "../TeamElement/TeamElement";
+import Divider from "@components/Divider/Divider";
+
+const TeamElementRow = ({
+  teamElements,
+}: {
+  teamElements: TeamElementProps[];
+}) => {
```
Comment (danielaprado) [inline fe/app/components/TeamElementRow/TeamElementRow.tsx:10]: I know it has only one prop but shouldn't we still have a `models.ts` file to go along with the standard we are using on the rest of the components?

**Review by catarinapm — APPROVED** [review]

**Pull request merged**